android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainActivity.kt

```diff
@@ -265,7 +264,12 @@ open class AppMainActivity : BaseMultiLanguageActivity(), QuestionnaireHandler,
           .await()
           ?.also {
             protoDataStore.writeLocationCoordinates(
-              LocationCoordinate(it.latitude, it.longitude, it.altitude, Instant.now()),
+              LocationCoordinate(
+                it.latitude,
+                it.longitude,
+                it.altitude,
+                Instant.now(),
```

Comment:
Use named parameters to improve readability. Apply the action by performing `Alt + Enter`.